UNRELEASED.yaml

```
@@ -1,4 +1,7 @@
 # UNRELEASED:
+FIXED:
+  - bpk-component-star-rating:
+    - The RTL interactive star rating had got a bit crazy - must be all this self-isolation. Think we've got it to settle down now.
```

```suggestion
    - Fixed a bug where the animation when selecting a star rating in RTL mode didn't appear correctly.
```

```
if (rightTime && rightPlace) {
  doAFunnyChangelogEntry()
} else {
  readTheRoomAndKeepItProfesh()
}
```

I know I sound like a killjoy but bear in mind these changelog entries are read by non-native English speakers from across the world, so the best thing we can do for our users is to keep them factual.